account_move_line_stock_info/hooks.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def post_init_hook(cr, registry):
+
+    """ INIT stock references in acount move line """
+    # FOR stock moves
+    cr.execute(
+        """
+        UPDATE account_move_line aml SET stock_move_id = am.stock_move_id
+        FROM account_move am
+        WHERE am.id = aml.move_id
+        AND am.stock_move_id IS NOT NULL;
```

Comment:
Maybe also:
```suggestion
        AND am.stock_move_id IS NOT NULL AND aml.stock_move_id IS NOT NULL;
```

Reply:
```suggestion
        AND am.stock_move_id IS NOT NULL AND aml.stock_move_id IS NULL;
```
hehe

Reply:
Opps yes!

Reply:
I don't agree with this. We want to make sure that we overwrite any previous value.

Reply:
> I don't agree with this. We want to make sure that we overwrite any previous value.

thanks for this explanation.
I'm not experienced enough on this part to approve or request a change, I let you as expert to "fight"